README.md

```diff
@@ -39,26 +39,20 @@ The ROS package can be installed and invoked by the following commands.
 ### Installation
 
 ```
-git clone https://github.com/jk-ethz/franka_lock_unlock
-catkin build franka_lock_unlock
+git clone https://github.com/jk-ethz/franka_lock_unlock.git
+colcon build
```

Comment:
Just to not build the entire workspace in case the user has a lot of packages in it.

```suggestion
colcon build --packages-up-to franka_lock_unlock
```